### DISCUSSION TITLE
Error training DeepMD NN on XYZ data converted with dpdata

### DISCUSSION THREAD

**Discussion by dmilardovich** [deepmodeling/dpdata#431] [Q&A] [answered]
Hi, I have a training dataset in .XYZ format (used to train GAP with the QUIP package). I used dpdata to convert it to deepmd_raw and deepmd_npy. It worked OK. However, when I try training a NN with DeepMD (nvnmd) on this data, I find the following error: 

f'Cannot feed value of shape {str(np_val.shape)} for Tensor '
ValueError: Cannot feed value of shape (3,) for Tensor d_sea_t_natoms:0, which has shape (4,)

 Could you please tell me what is wrong with this data? I am sending attached the original .XYZ file and the training file. Thanks! 
[dpdata_error.zip](https://github.com/deepmodeling/dpdata/files/10921470/dpdata_error.zip)

**Comment by LiuGroupHNU** [comment, marked as answer]
Hi.
I see your dataset only contains one type of element ("Ge"). However, your input file contains two elements ('"sel": [80, 80],').
You can change it to `"sel": [80]`.
Good luck!

**Comment by dmilardovich** [comment]
Hi, you are right, "sel" was set to "[80], [80]" from the previous test for GeTe. I changed it to "[80]" and it is working now for Ge. Thanks! :)

**Comment by composites123** [comment]
Hello, I had a problem converting GAP's xyz format training set. When converting the data using dpdata's MultiSystems.from_file command, I found that the number of frames in the training set could not be accurately identified. It seems that dpdata combines frames with the same atomic type and number into the same frame, do you have this problem? If so, how did you resolve it?
